core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -1140,27 +1140,447 @@ public enum SqlKind {
 
-  /** The {@code ST_MakeLine} function that makes a line. */
-  ST_MAKE_LINE,
+  /** The {@code ST_PointFromWKB} function. */
+  ST_POINTFROMWKB,
 
-  /** The {@code ST_Contains} function that tests whether one geometry contains
-   * another. */
-  ST_CONTAINS,
+  /** The {@code ST_PointN} function. */
+  ST_POINTN,
+
+  /** The {@code ST_PointOnSurface} function. */
+  ST_POINTONSURFACE,
+
+  /** The {@code ST_PolyFromText} function. */
+  ST_POLYFROMTEXT,
+
+  /** The {@code ST_PolyFromWKB} function. */
+  ST_POLYFROMWKB,
+
+  /** The {@code ST_Polygonize} function. */
+  ST_POLYGONIZE,
+
+  /** The {@code ST_PrecisionReducer} function. */
+  ST_PRECISIONREDUCER,
+
+  /** The {@code ST_ProjectPoint} function. */
+  ST_PROJECTPOINT,
+
+  /** The {@code ST_ReducePrecision} function. */
+  ST_REDUCEPRECISION,
+
+  /** The {@code ST_Relate} function. */
+  ST_RELATE,
+
+  /** The {@code ST_RemoveHoles} function. */
+  ST_REMOVEHOLES,
+
+  /** The {@code ST_RemovePoint} function. */
+  ST_REMOVEPOINT,
+
+  /** The {@code ST_RemoveRepeatedPoints} function. */
+  ST_REMOVEREPEATEDPOINTS,
+
+  /** The {@code ST_Reverse} function. */
+  ST_REVERSE,
+
+  /** The {@code ST_Rotate} function. */
+  ST_ROTATE,
+
+  /** The {@code ST_SRID} function. */
+  ST_SRID,
+
+  /** The {@code ST_Scale} function. */
+  ST_SCALE,
+
+  /** The {@code ST_SetSRID} function. */
+  ST_SETSRID,
+
+  /** The {@code ST_Simplify} function. */
+  ST_SIMPLIFY,
+
+  /** The {@code ST_SimplifyPreserveTopology} function. */
+  ST_SIMPLIFYPRESERVETOPOLOGY,
+
+  /** The {@code ST_Snap} function. */
+  ST_SNAP,
+
+  /** The {@code ST_Split} function. */
+  ST_SPLIT,
+
+  /** The {@code ST_StartPoint} function. */
+  ST_STARTPOINT,
+
+  /** The {@code ST_SymDifference} function. */
+  ST_SYMDIFFERENCE,
+
+  /** The {@code ST_ToMultiLine} function. */
+  ST_TOMULTILINE,
+
+  /** The {@code ST_ToMultiPoint} function. */
+  ST_TOMULTIPOINT,
+
+  /** The {@code ST_ToMultiSegments} function. */
+  ST_TOMULTISEGMENTS,
+
+  /** The {@code ST_Touches} function. */
+  ST_TOUCHES,
+
+  /** The {@code ST_Transform} function. */
+  ST_TRANSFORM,
+
+  /** The {@code ST_Translate} function. */
+  ST_TRANSLATE,
+
+  /** The {@code ST_UnaryUnion} function. */
+  ST_UNARYUNION,
+
+  /** The {@code ST_UnaryUnion} function. */
+  ST_UNION,
+
+  /** The {@code ST_Within} function. */
+  ST_WITHIN,
+
+  /** The {@code ST_X} function. */
+  ST_X,
+
+  /** The {@code ST_XMax} function. */
+  ST_XMAX,
+
+  /** The {@code ST_XMin} function. */
+  ST_XMIN,
+
+  /** The {@code ST_Y} function. */
+  ST_Y,
+
+  /** The {@code ST_YMax} function. */
+  ST_YMAX,
+
+  /** The {@code ST_YMin} function. */
+  ST_YMIN,
+
+  /** The {@code ST_Z} function. */
+  ST_Z,
+
+  /** The {@code ST_ZMax} function. */
+  ST_ZMAX,
+
+  /** The {@code ST_ZMin} function. */
+  ST_ZMIN,
 
   /** The {@code Hilbert} function that converts (x, y) to a position on a
    * Hilbert space-filling curve. */
   HILBERT,
 
+  ST_MAKE_LINE,
```

Comment:
Are these obsolete or not?

Reply:
I believe we can remove them. ST_MAKE_LINE is ST_MAKELINE and all results returned by search engines for ST_POINT3 are related to ST_POINT.